src/wp-includes/block-template-utils.php

```diff
@@ -837,3 +872,55 @@ function block_header_area() {
 function block_footer_area() {
 	block_template_part( 'footer' );
 }
+
+/**
+ * Creates an export of the current templates and
+ * template parts from the site editor at the
+ * specified path in a ZIP file.
+ *
+ * @since 5.9.0
+ *
+ * @return WP_Error|string Path of the ZIP file or error on failure.
+ */
+function wp_generate_block_templates_export_file() {
+	if ( ! class_exists( 'ZipArchive' ) ) {
+		return new WP_Error( __( 'Zip Export not supported.' ) );
+	}
```

Comment:
Is the UI hidden if `ZipArchive` is unavailable? 

This line will be needed even if it is but I just want to make sure it is.

Reply:
Currently, UI is always visible since we will have to make a new request to check if `ZipArchive` is available.

Reply:
In my testing, the export feature did not fail gracefully. 

https://user-images.githubusercontent.com/237508/144089829-a2815cfd-4e42-4b88-84ee-079152b91839.mov